clients/js/vaa.ts

```diff
@@ -209,6 +212,68 @@ export function sign(signers: string[], vaa: VAA<Payload>): Signature[] {
     })
 }
 
+const kmsClient = new KeyManagementServiceClient({
+    keyFile: "/tmp/credentials-file.json"
```

Comment:
Question: is this file used for testing only?

Reply:
This was used for generating the docker image needed to test KMS integration in the cloud (VAAs for token bridge registration, etc need to be signed using KMS). Changing it to be configurable now.

Reply:
Let's not put credential files in docker image 😅

Reply:
Key itself will not be in docker image :lock_with_ink_pen: 
The generated VAAs using KMS will be in the docker images.

Reply:
Per our discussion, we should remove all of the KMS code from `clients/js`.

Reply:
Removed all KMS related code from clients/js.